client/src/pages/NeedHelp.js

```diff
@@ -108,6 +109,10 @@ const Step3 = (props) => {
         Question {props.currentStep}/{props.totalSteps}
       </WizardProgress>
       <StepTitle>What is your email address?</StepTitle>
+      <StyledDiv>
+        We respect your privacy. Please read our{" "}
```

Comment:
I know this was added by the linter, but the `{" "}` shouldn't be here.

Reply:
Ohhh actually this was intentional, I reviewed this before Rixio committed. If you remove {" "}, then there won't be space between the our and Link...